cmd/state-svc/main.go

```diff
@@ -186,6 +186,7 @@ func runForeground(cfg *config.Instance, an *anaSync.Client, auth *authenticatio
 	logFile := logging.FilePath()
 	logging.Debug("Logging to %q", logFile)
 	logging.StartRotateLogTimer()
+	defer logging.StopRotateLogTimer()
```

Comment:
Can we make `StartRotateLogTimer()` return the stop function and invoke it here? 

Realize the logging lib is riddled with globals already, but still rather avoid adding to that technical debt.